crates/types/src/config/networking.rs

```diff
@@ -94,7 +94,7 @@ impl Default for NetworkingOptions {
             http2_keep_alive_interval: Duration::from_secs(5).into(),
             http2_keep_alive_timeout: Duration::from_secs(5).into(),
             http2_adaptive_window: true,
-            num_concurrent_connections: NonZeroUsize::new(13).unwrap(),
+            num_concurrent_connections: NonZeroUsize::new(3).unwrap(),
```

Comment:
How did you choose `3` as the magic number?

Reply:
I'll revert this for now, my end goal is to reduce the number of streams between nodes but I am holding until the rest of the changes are ready.